backend/app/services/execution/agents/image/intent_analyzer.py

```
@@ -0,0 +1,233 @@
+            if block.get("type") == "image":
+                attachment_ids = block.get("image_attachment_ids", [])
+                if attachment_ids:
+                    attachment_id = attachment_ids[0]
+                    db = SessionLocal()
+                    try:
+                        context = context_service.get_context_optional(
+                            db=db, context_id=attachment_id
+                        )
+                        if context and context.status == "ready":
+                            return context_service.build_attachment_url(attachment_id)
+                    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)